composer.json

```
@@ -17,7 +17,7 @@
         "php": "^8.1.0",
         "laminas/laminas-diactoros": "^3.0",
         "laravel/framework": "^10.10.1|^11.0",
-        "laravel/prompts": "^0.1.24",
+        "laravel/prompts": "^0.1.24|^0.2",
```

```suggestion
        "laravel/prompts": "^0.1.24|^0.2.0|^0.3.0",
```

We are currently in discussion to release `0.3.0` so adding this and marking it as draft for now. I will update once we have definitive plan for week 1 of October releases.